metrics/storage/README.md

```diff
@@ -0,0 +1,10 @@
+# Storage metrics tests
+
+This section includes metrics tests focused on measure storage bandwith under different
+conditions/configuration where could be executed different I/O operation such as: read,
+write, random read etc.
+However the use of these should be complemented with more tools in order to find the hot spot.
+
+- `fio_job.sh`: measures the bandwidth measured by a specif I/O operation (read, write,
+   random-read, etc), and using a default configuration, the operations and configuration
```

Comment:
break this into two sentences. If you are saying that there is a default, it may be best to re-work the sentence order:

Measures storage bandwidth. This test is executed with a specific I/O operation (read, write, random-read, etc) using a default configuration. However, the default operations and configuration can be overwritten.